### PR TITLE
Upload Android APKs from Github Pull Requests.

### PR DESCRIPTION
**Pull request by sigmabeta**
Folks have been complaining they can't test Android changes because the PRs don't get uploaded anywhere. They were being built, but not made available.

This hopefully fixes that. I'm not a buildbot expert though, so I probably messed something up.

### CODE REVIEW

File: buildbot/master.cfg [hunk 1]
```diff
@@ -576,6 +577,13 @@ def make_android_package(mode="normal"):
                                descriptionDone="Gradle",
                                haltOnFailure=True))
 
+        master_filename = WithProperties("/srv/http/dl/prs/dolphin-%s.apk", "branchname")
+        url = WithProperties("http://dl.dolphin-emu.org/prs/dolphin-%s.apk", "branchname")
+        
+        f.addStep(FileUpload(slavesrc="Source/Android/app/build/outputs/apk/app-arm_64-release.apk",
```
Comment (delroth) [inline buildbot/master.cfg]: Move this out of the if? It's the same on both sides AFAICT.

Comment (BhaaLseN) [inline buildbot/master.cfg]: Difference is the shortrev; which probably makes sense to keep?

Comment (delroth) [inline buildbot/master.cfg]: I was talking about addStep only.

Comment (BhaaLseN) [inline buildbot/master.cfg]: The rest is still the same, except for the shortrev. What other modes are there anyways except "normal"? It'd probably make sense to do the same for both branches?

Comment (sigmabeta) [inline buildbot/master.cfg]: If 'shortrev' is important information (I dunno what it is in case of a PR) then I'm happy to make the change.

Comment (sigmabeta) [inline buildbot/master.cfg]: The modes are 'normal', 'pr', and 'wip'. Not sure what 'wip' is used for, though.

Comment (BhaaLseN) [inline buildbot/master.cfg]: "wip" is for buildbot-try, but you'd need to be authenticated to be allowed to use that.
Only thing left to note is probably the format of the file names for the PR-folder, which looks different for all the other build types (wrt sorting when browsing the page), but +/-0 on keeping it as-is.

**Comment by Parlane** [comment]
elif wip:
        master_filename = WithProperties("/srv/http/dl/wips/%%s-dolphin-%%s-%%s-%s.7z" % fn_arch, "author", "branchname", "shortrev")
        url = WithProperties("http://dl.dolphin-emu.org/wips/%%s-dolphin-%%s-%%s-%s.7z" % fn_arch, "author", "branchname", "shortrev")
   

wip builds need to go in their own folder.

**Comment by sigmabeta** [comment]
Thanks for the heads up, I'll make this change in the morning

**Comment by Parlane** [comment]
Yep so look at make_dolphin_win_build as you will need the:

    mode = mode.split(",")
    debug = "debug" in mode
    wip = "wip" in mode
    pr = "pr" in mode

**Pull request merged**